server/test/integration/com/thoughtworks/go/config/GoConfigMigrationIntegrationTest.java

```diff
@@ -1450,6 +1450,24 @@ public void shouldConvertPluginToPluginIdOnConfigRepoAsPartOfMigration95() throw
         assertThat(migratedContent, containsString("pluginId=\"json.config.plugin\""));
     }
 
+
+    @Test
+    public void shouldNotSupportedUncesseryMaterialFieldsAsPartOfMigration98() throws Exception {
+        String configXml = "<cruise schemaVersion='97'>" +
+                "<config-repos>\n" +
+                "   <config-repo pluginId=\"json.config.plugin\" id=\"config-repo-1\">\n" +
+                "     <git url=\"https://github.com/tomzo/gocd-json-config-example.git\" dest=\"dest\"/>\n" +
+                "   </config-repo>\n" +
+                "</config-repos>" +
+                "</cruise>";
+
+        try {
+            migrateXmlString(configXml, 97);
+        } catch (InvocationTargetException e) {
```

Comment:
Consider using `@Rule` for this? The test does not verify that an exception is thrown.